src/Pages/sangjunkim/Login/Login.js

```diff
@@ -0,0 +1,233 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+
+const moreInfo = [
```

Comment:
이렇게 변하지 않는 데이터는 대문자_대문자로 적어주세요!
```suggestion
const MORE_INFO = [
```

Reply:
수정하였습니다.